adapters/cpmstar/usersync_test.go

```diff
@@ -0,0 +1,20 @@
+package cpmstar
+
+import (
+	"testing"
+	"text/template"
+
+	"github.com/prebid/prebid-server/privacy"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCpmstarSyncer(t *testing.T) {
+	temp := template.Must(template.New("sync-template").Parse("https://server.cpmstar.com/usersync.aspx?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirectUri=http%3A%2F%2Flocalhost:8000%2Fsetuid%3Fbidder%3Dcpmstar%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26us_privacy%3D{{.USPrivacy}}%26uid%3D%24UID"))
```

Comment:
We recently updated the standard style of these tests to improve readability. Please consider using the update style, ex:
https://github.com/prebid/prebid-server/blob/master/adapters/appnexus/usersync_test.go